### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

### CODE REVIEW

File: service/api/conversions.go [hunk 1]
```diff
@@ -210,7 +210,7 @@ func ConvertFunction(fn *proc.Function) *Function {
 	}
 
 	// fn here used to be a *gosym.Func, the fields Type and GoType below
-	// corresponded to the omonymous field of gosym.Func. Since the contents of
+	// corresponded to the anonymous field of gosym.Func. Since the contents of
```
Comment (jsoref) [inline service/api/conversions.go]: I'm not at all sure about this

File: pkg/proc/types.go [hunk 2]
```diff
@@ -799,7 +799,7 @@ func constructTypeForKind(kind int64, bi *BinaryInfo) (*godwarf.StructType, erro
 	}
 	prtyp := pointerTo(rtyp, bi.Arch)
 
-	uintptrtyp, err := bi.findType("uintptr")
+	uintptrtype, err := bi.findType("uintptr")
```
Comment (jsoref) [inline pkg/proc/types.go]: I'm pretty sure this is a reasonable change

File: pkg/proc/variables.go [hunk 3]
```diff
@@ -625,18 +625,18 @@ func (scope *EvalScope) PackageVariables(cfg LoadConfig) ([]*Variable, error) {
 	var vars []*Variable
 	reader := scope.DwarfReader()
 
-	var utypoff dwarf.Offset
+	var utypeoff dwarf.Offset
```
Comment (jsoref) [inline pkg/proc/variables.go]: I'm pretty sure this is a reasonable change

**Comment by derekparker** [comment]
LGTM but please remove any changes to vendored files.

**Comment by jsoref** [comment]
@derekparker: sorry about that, I had removed the vendored files when I did my spelling work, and then prepared to publish, but then I went back and reviewed the codebase for `it's` -- which isn't actually a spelling thing, but I wasn't paying attention to the file paths....

File: service/api/conversions.go (hunk 1 above)
Comment (aarzilli) [reply service/api/conversions.go]: it's homonymous

File: pkg/proc/variables.go (hunk 3 above)
Comment (aarzilli) [reply pkg/proc/variables.go]: There's a lot of places where we use typ instead of type, I would leave those as they are for consistency.

**Comment by derekparker** [comment]
I'm good with this after @aarzilli suggestions.

**Comment by jsoref** [comment]
@derekparker: dropped (anonymous was split) + rebased

**Comment by aarzilli** [comment]
Thank you.

**Pull request merged**